utils.js

```diff
@@ -0,0 +1,29 @@
+/*******************************************************************************
+
+    publicsuffixlist.js - an efficient javascript implementation to deal with
+    Mozilla Foundation's Public Suffix List <http://publicsuffix.org/list/>
+
+    Copyright (C) 2013-present Raymond Hill
+
+    License: pick the one which suits you:
+      GPL v3 see <https://www.gnu.org/licenses/gpl.html>
+      APL v2 see <http://www.apache.org/licenses/LICENSE-2.0>
+
+*/
+
+/*! Home: https://github.com/gorhill/publicsuffixlist.js -- GPLv3 APLv2 */
+
+/******************************************************************************/
+
+import { readFile } from 'fs/promises';
+
+/******************************************************************************/
+
+export async function minimalNodeFetch(url) {
```

Comment:
This utility function is for passing to `enableWASM()` in case the caller doesn't want to use a full implementation like [node-fetch](https://www.npmjs.com/package/node-fetch). ubo-core will probably use this. Other Node.js clients may want to use node-fetch.

Reply:
On second thoughts, this is not necessary. ubo-core is the only client that would need this. It should go there. In `README.md` we can use node-fetch as an example.